src/Files.App/UserControls/TabBar/TabBar.xaml.cs

```diff
@@ -51,6 +55,14 @@ public bool AllowTabsDrag
 		public Rectangle DragArea
 			=> DragAreaRectangle;
 
+		private GridLength _titleBarWidth;
+
+		public GridLength TitleBarWidth
+		{
+			get => _titleBarWidth;
+			set => _titleBarWidth = new((value.Value / App.AppModel.AppWindowDPI) + _gap);
```

Comment:
Assuming the `TitleBar.Inset` property isn't null, I think it's already DPI aware and we can use the value as is.

Reply:
No, I tested with 1.25 dpi and the value is not recalculated.